<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Pedido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Nuevo Pedido</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Codigo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cantidad:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Lista de Productos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="rowCount">
            <number>1</number>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <row/>
           <column>
            <property name="text">
             <string>Codigo</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cantidad</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Costo unitario</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Costo total</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Cantidad</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>TOTAL</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="enviarPedido">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
           </property>
           <property name="centerButtons">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>textEdit</zorder>
        <zorder>tableWidget</zorder>
        <zorder>horizontalLayoutWidget</zorder>
        <zorder>horizontalLayoutWidget_2</zorder>
        <zorder>pushButton_2</zorder>
        <zorder>enviarPedido</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>tableWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionConfiguracion"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuFacturas">
    <property name="title">
     <string>Factura</string>
    </property>
    <addaction name="actionNueva"/>
    <addaction name="actionBuscar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuPedido">
    <property name="title">
     <string>Pedido</string>
    </property>
    <addaction name="actionNuevo"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuPedido"/>
   <addaction name="menuFacturas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfiguracion">
   <property name="text">
    <string>Configuracion</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionNueva">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
